lib/vanagon/platform.rb

```diff
@@ -237,8 +237,12 @@ def [](key)
     # @param target_repo [String] optional repo target for built packages defined
     #   at the project level
     # @return [String] relative path to where packages should be output to
-    def output_dir(target_repo = "")
-      @output_dir ||= File.join(@os_name, @os_version, target_repo, @architecture)
+    def output_dir(target_repo = nil)
+      if target_repo.nil?
+        @output_dir ||= "#{@os_name}-#{@os_version}-#{@architecture}"
+      else
+        @output_dir ||= File.join(@os_name, @os_version, target_repo, @architecture)
```

Comment:
yep, okay, I see now. I thought I had read the comments before starting to look at the code, but I think I mixed up which PRs I had read. Sorry about that!